<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenTorrentDialog</class>
 <widget class="QDialog" name="OpenTorrentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DIALOG_OPEN_TORRENT</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../Resources/ltorrent.qrc">
    <normaloff>:/icons/app.ico</normaloff>:/icons/app.ico</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="tbMenu">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../Resources/ltorrent.qrc">:/icons/app.ico</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LTitle">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>DIALOG_OPEN_TORRENT</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_centralWidget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="6">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>OPEN_TORRENT_SAVE_PATH</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QComboBox" name="GroupComboBox"/>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="pathEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>DISCRIBTION_LABEL</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="browseButton">
           <property name="text">
            <string>OPEN_TORRENT_BROWSE_BUTTON</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>OPEN_TORRENT_INFO</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="labelNameTitile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>OPEN_TORRENT_INFO_NAME</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="labelNameData">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="2">
          <widget class="QLabel" name="labelCommentTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>OPEN_TORRENT_INFO_DESCRIBTION</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelComentData">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1" rowspan="2">
          <widget class="QLabel" name="labelSizeData">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelSizeTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>SIZE</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="loaderGifLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="loaderTextLabel">
           <property name="text">
            <string>RECIVING_METADATA</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="4">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="setSequntialCheckBox">
             <property name="toolTip">
              <string>START_SEQUENTIAL_TOOLTIP</string>
             </property>
             <property name="text">
              <string>SET_SEQUNTIAL</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="setSeedModeCheckBox">
             <property name="toolTip">
              <string>START_SEEDING_TOOLTIP</string>
             </property>
             <property name="text">
              <string>START_SEEDING</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="setPausedCheckBox">
             <property name="toolTip">
              <string>START_PAUSED_TOOLTIP</string>
             </property>
             <property name="text">
              <string>START_PAUSED</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="setSuperSeedCheckBox">
             <property name="toolTip">
              <string>START_WITH_SUPER_SEED</string>
             </property>
             <property name="text">
              <string>START_WITH_SUPER_SEED</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0" colspan="4">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="markAllButton">
             <property name="text">
              <string>MARK_ALL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="unmarkAllButton">
             <property name="text">
              <string>UNMARK_ALL</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0" colspan="4">
          <widget class="QTreeView" name="torrentFilesTreeView">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>266</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="cancelButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>OPEN_TORRENT_CANCEL_BUTTON</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="yesButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>OPEN_TORRENT_ADD_BUTTON</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Resources/ltorrent.qrc"/>
  <include location="../../../Resources/ltorrent.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>OpenTorrentDialog</receiver>
   <slot>BrowseButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>OpenTorrentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yesButton</sender>
   <signal>clicked()</signal>
   <receiver>OpenTorrentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GroupComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>OpenTorrentDialog</receiver>
   <slot>ChangeGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OpenTorrentDialog</receiver>
   <slot>OnPathChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>markAllButton</sender>
   <signal>clicked()</signal>
   <receiver>OpenTorrentDialog</receiver>
   <slot>OnCheckAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unmarkAllButton</sender>
   <signal>clicked()</signal>
   <receiver>OpenTorrentDialog</receiver>
   <slot>OnUncheckAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CancelButton()</slot>
  <slot>BrowseButton()</slot>
  <slot>AccepTorrent()</slot>
  <slot>ChangeGroup()</slot>
  <slot>DownloadMetadataCompleted()</slot>
  <slot>OnPathChanged(QString)</slot>
  <slot>OnCheckAll()</slot>
  <slot>OnUncheckAll()</slot>
 </slots>
</ui>
